wcfsetup/install/files/lib/util/HeaderUtil.class.php

```diff
@@ -48,7 +48,13 @@ public static function setCookie($name, $value = '', $expire = 0) {
 			$cookieDomain = explode(':', $cookieDomain, 2)[0];
 		}
 		
-		@header('Set-Cookie: '.rawurlencode(COOKIE_PREFIX.$name).'='.rawurlencode($value).($expire ? '; expires='.gmdate('D, d-M-Y H:i:s', $expire).' GMT; max-age='.($expire - TIME_NOW) : '').'; path=/'.($addDomain ? '; domain='.$cookieDomain : '').(RouteHandler::secureConnection() ? '; secure' : '').'; HttpOnly', false);
+		$sameSitePolicy = '';
+		$sameSiteOption = COOKIE_SAME_SITE_ATTRIBUTE;
```

Comment:
continuing my comment in the fresh diff, because it is being collapsed otherwise:

> Well, it's only reassigned if $sameSiteOption is not 'none'. Otherwise it never gets assigned, and I believe it is a good practice to always declare variables that are used.

I misread one of the variable names as both being the same. Do not use a temporary variable just for the `if` expression then (i.e. remove this line).

Reply:
Ah, yeah. No I realize that it's not needed anymore. It was used previously, because - in my first attempt - I checked with empty(). And that requires its parameter to be a variable. Gonna remove the variable of course.